package.json

```diff
@@ -8,7 +8,7 @@
     "puglint": "pug-lint .",
     "lint": "npm run eslint && npm run puglint",
     "mocha": "mocha --timeout 15000 --slow 500 --exit -i -g \"functional\" -R dot ./tests/",
-    "mocha:suite": "mocha --timeout 15000 --slow 500 --exit -R dot ./tests/",
+    "mocha:suite": "mocha --timeout 15000 --slow 500 --exit ./tests/",
```

Comment:
We should probably revert this line before releasing. It makes the output insanely long. Another option might be `-R tap`, which is still verbose, but should be about 1/3 the output.